drivers/sensor/grow_r502a/Kconfig

```diff
@@ -0,0 +1,59 @@
+# GROW_R502A hzgrow Fingerprint sensor Configuration options
+
+# Copyright (c) 2021 Linumiz
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig GROW_R502A
+	bool "GROW_R502A Fingerprint Sensor"
```

Comment:
Update to use new Kconfig.dts (generated) pattern.  See other drivers/sensors/*/Kconfig for examples:

```
default y
depends on DT_HAS_HZGROW_GROW_R502A_ENABLED
```

Reply:
fixed.